### DISCUSSION TITLE
Exploding the System in LAMMPS Simulation

### DISCUSSION THREAD

**Discussion by albrono** [deepmodeling/deepmd-kit#1187] [Q&A]
Hello,
I have trained a model for the silicon system. All of the loss functions converged, and the model accuracy on the test set was high. Then I ran a tension test simulation with about 500 atoms with the frozen model, giving a logical stress-strain curve. However, when I increased the atoms to 5000, the simulation broke down with an exploded system.
What is the problem, and how can I solve it?
Thanks,

**Comment by njzjz** [comment]
Do they have the same density? I think the main reason is that the training data is not enough.

**Reply by albrono** [reply]
Yes, they are the same. 
I have about 5000 frames for the training set, Is it enough?
What AIMD simulation should be considered for the tension test in lammps?
